packages/react-server/src/ReactFlightServer.js

```diff
@@ -1573,6 +1562,26 @@ function renderElement(
   } else if (type === REACT_FRAGMENT_TYPE && key === null) {
     // For key-less fragments, we add a small optimization to avoid serializing
     // it as a wrapper.
+    if (__DEV__ && enableOwnerStacks && validated === 2) {
+      // Create a fake owner node for the error stack.
+      const componentDebugInfo = ({
+        name: 'Fragment',
+        env: (0, request.environmentName)(),
+        owner: task.debugOwner,
+      }: ReactComponentInfo);
+      if (enableOwnerStacks) {
```

Comment:
do you need this?

Reply:
or were you supposed to remove the flag gate above?

Reply:
oh yea